packages/hardhat-core/test/internal/hardhat-network/provider/modules/eth/methods/call.ts

```diff
@@ -603,7 +603,8 @@ describe("Eth module", function () {
             );
           });
 
-          it("should throw an error because an invalid storage key is used", async function () {
+          // TODO: https://github.com/NomicFoundation/edr/issues/104
+          it.skip("should throw an error because an invalid storage key is used", async function () {
```

Comment:
This one returns a value instead of throwing. I think we should fix that before merging.

Reply:
Fixed by https://github.com/NomicFoundation/hardhat/pull/4666/commits/2aa30168e8246a9e76cb4a64f872d13ee8ece3dc